test/tests.sh

```diff
@@ -497,8 +533,13 @@ test_config()
 
     wd -q --config "$arg_config" add
 
-    assertEquals 1 "$(wcl < "$arg_config")"
+    # confirm $WD_CONFIG was unchanged
     assertEquals "$wd_config_lines" "$(total_wps)"
+    assertEquals "$wd_config_lines" 0
+
+    # confirm --config was changed
+    assertEquals 1 "$(wcl < "$arg_config")"
+    assertEquals 1 "$(WD_CONFIG=$arg_config total_wps)"
```

Comment:
No changes to the asserts in this PR or the linked, but it's added for clarity as part of #141 🔬